src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -456,16 +461,43 @@ public function deleteAction($id)
      */
     public function deleteVersionAction(Request $request, $id, $version)
     {
-        $locale = $this->getRequestParameter($request, 'locale', true);
-        $media = $this->mediaManager->getById($id, $locale);
+        if (!\method_exists($this->mediaManager, 'removeFileVersion')) {
```

Comment:
not sure if this can happen. the constructor argument has a `MediaManagerInterface` typehint, so `$this->mediaManager` must be a `MediaManagerInterface`. i think php will throw an error if you dont implement a method of an interface, so i dont think this condition can be true🙂

Reply:
The `MediaManagerInterface` doesn't define this method, it's just defined as phpdoc